payments-core/src/main/java/com/stripe/android/payments/paymentlauncher/PaymentLauncher.kt

```diff
@@ -42,20 +43,40 @@ interface PaymentLauncher {
     }
 
     companion object {
+        /**
+         * Create a [PaymentLauncher] instance with [ComponentActivity].
+         *
+         * This API registers an [ActivityResultLauncher] into the [ComponentActivity],  it needs
+         * to be called before the [ComponentActivity] is created.
+         */
         fun create(
             activity: ComponentActivity,
             publishableKey: String,
             stripeAccountId: String? = null,
             callback: PaymentResultCallback
         ) = PaymentLauncherFactory(activity, callback).create(publishableKey, stripeAccountId)
 
+
+        /**
+         * Create a [PaymentLauncher] instance with [Fragment].
+         *
+         * This API registers an [ActivityResultLauncher] into the [Fragment]'s hosting Activity, it
+         * needs to be called before the [Fragment] is created.
+         */
         fun create(
             fragment: Fragment,
             publishableKey: String,
             stripeAccountId: String? = null,
             callback: PaymentResultCallback
         ) = PaymentLauncherFactory(fragment, callback).create(publishableKey, stripeAccountId)
 
+        /**
+         * Create a [PaymentLauncher] used for Jetpack Compose.
+         *
+         * This API uses Compose specific API [rememberLauncherForActivityResult] to register a
+         * [ActivityResultLauncher] into current activity, it should be called as part of Compose
+         * initialization path.
+         */
```

Comment:
Do you have an example of this in compose?

Reply:
I have test it in [LaunchPaymentSheetWithComposeActivity](https://github.com/stripe/stripe-android/blob/15dc2852f3ceca37508f69adbec705d7d5ec6806/paymentsheet-example/src/main/java/com/stripe/android/paymentsheet/example/activity/LaunchPaymentSheetWithComposeActivity.kt#L14), but since this does not belong to `PaymentSheet`, will start a dedicate PR to demonstrate it the example app. Added it in the next steps in the PR description